src/server/worldserver/worldserver.conf.dist

```diff
@@ -2363,6 +2363,14 @@ Rate.Drop.Item.Artifact         = 1
 Rate.Drop.Item.Referenced       = 1
 Rate.Drop.Money                 = 1
 
+#    RewMoneyMaxLevel
+#        Description: Allows to further tweak the amount of extra money
+#                     rewarded by quests when the player is at MaxPlayerLevel.
+#                     NOTE: the final amount will also affected by Rate.Drop.Money
+#        Default:     1
+
+Rate.RewMoneyMaxLevel = 1
```

Comment:
the description doesn't explain anything and is extremely confusing

in fact I have no idea what this feature is about

if you want it to be proportional to the max level, then it is just logic and should not be enablable or not, it should just be coded like that and basta :)